caesars-palace/blocks/footer/footer.js

```diff
@@ -9,7 +9,7 @@ export default async function decorate(block) {
   const cfg = readBlockConfig(block);
   block.textContent = '';
 
-  const footerPath = cfg.footer || '/footer';
+  const footerPath = cfg.footer || '/caesars-palace/footer';
```

Comment:
Can't we define `/caesars-palace/` as a prefix somewhere and reuse it everywhere? So it it would need changes, we change only at one place (also prevents typos).